src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -75,7 +75,7 @@ public enum BailCaseFieldDefinition {
     APPEAL_REFERENCE_NUMBER(
         "appealReferenceNumber", new TypeReference<String>(){}),
     HAS_PREV_BAIL_APPLICATION(
-        "hasPreviousBailApplication", new TypeReference<String>(){}),
+        "hasPreviousBailApplication", new TypeReference<YesOrNo>(){}),
```

Comment:
I think this field doesn't get used.
I think it's already using HAS_PREVIOUS_BAIL_APPLICATION

Reply:
ah you're right, this was for the other part of the ticket which was removing the related case fields if they didn't have a previous application. I'll revert this change.